server/services/flightRecord.js

```diff
@@ -1,17 +1,83 @@
 const FlightRecord = require("../models/flightRecordsSchema");
 const Airplane = require("../models/airplaneSchema");
 const Flight = require("../models/flightSchema");
+const Ticket = require("../models/ticketSchema");
+
+function generateTickets(
+  bsTickets,
+  stTickets,
+  ecTickets,
+  bsPrice,
+  stPrice,
+  ecPrice,
+  id
+) {
+  const tickets = [];
+
+  for (let i = 0; i < bsTickets; i++) {
```

Comment:
I dont like this approach. What if we'll need to add 4-th class? 5-th? You'll need to add new the same code here. We need code which won't depend on count of classes. Do you understand? Maybe it's better to use objects to discribe groups of classes. Something like { class: ..., countOfSeats: ... } (maybe extra field price). In that case it enougth to have one handler.

Reply:
fixed it